.github/workflows/regression-tests.yml

```diff
@@ -32,64 +29,16 @@ jobs:
       - name: Configure & Build
         run: |
           apt update && apt install -y pip jq
-          cmake -B ${GITHUB_WORKSPACE}/build -DCMAKE_BUILD_TYPE=${{matrix.build-type}} -GNinja \
+          cmake -B ${GITHUB_WORKSPACE}/build-opt -DCMAKE_BUILD_TYPE=${{matrix.build-type}} -GNinja \
```

Comment:
the reason we have not called it opt because we have a matrics that build debug build as well. 
what's the reason you had to rename?

Reply:
So I don't have to pass the build directory as an argument to the action. Since there are dependencies, I will simply add a parameter to the action